src/SFA.DAS.LevyTransferMatching/Interfaces/ILevyTransferMatchingService.cs

```diff
@@ -15,7 +17,7 @@ public interface ILevyTransferMatchingService
         Task<IEnumerable<ReferenceDataItem>> GetSectors();
         Task<IEnumerable<ReferenceDataItem>> GetJobRoles();
         Task<PledgeReference> CreatePledge(Pledge pledge);
-		Task<IEnumerable<Pledge>> GetPledges();
+		Task<GetPledgesResponse> GetPledges(GetPledgesRequest getPledgesRequest);
```

Comment:
`getPledgesRequest` should be named `request` for consistency with the other method params

Reply:
Good point; updated both interface and concrete implementation.